src/server.c

```diff
@@ -80,7 +81,11 @@ int dqlite__init(struct dqlite_node *d,
 			 "config__init(rv:%d)", rv);
 		goto err;
 	}
+	d->config.disk = disk_mode;
 	rv = VfsInit(&d->vfs, d->config.name);
```

Comment:
This line is not used if disk_mode is set so we might guard it in the conditional below.

Reply:
dqlite_vfs_enable_disk relies on VfsInit being called first (to allocate `struct vfs`).

Reply:
You are right